### DISCUSSION TITLE
YAML Build Pipelines with Universal Packages not working for release notes generation

### DISCUSSION THREAD

**Discussion by schmolfi** [rfennell/AzurePipelines#1498] [General]
Hello,

we currently try to move everything from classic release pipelines to YAML.
In our classic release pipelines, we use universal packages as artifact resources. The problem is, that universal packages can not be integrated as pipeline resources in YAML.
Therefore, we download particular universal packages with the following YAML task.

            - task: UniversalPackages@0
              displayName: Download universal package ${{ artifact.value.universalPackage }}
              inputs:
                command: download
                downloadDirectory: $(Build.ArtifactStagingDirectory)/${{ artifact.value.universalPackage }}
                feedsToUse: internal
                vstsFeed: ****
                vstsFeedPackage: ${{ artifact.value.universalPackage }}
                vstsPackageVersion: ${{ artifact.value.version }}
                
The problem right now is, that task: richardfennellBM.BM-VSTS-XplatGenerateReleaseNotes.XplatGenerate-Release-Notes.XplatGenerateReleaseNotes@3 doesn't find the downloaded packages. Is there any option to configure that?

**Comment by rfennell** [comment]
No there is no way to achieve that, my task as you noted
- Looks for WI/CS changes single the last successful build
- Manual Test
- Associated pipeline resources

The contents of the universal package downloaded via a task does not provide me with a way to know if its presense

All I can suggest is you write a custom handlebars  extensions for my task, as [documented  in the WIKI](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task#handlebar-extensions) that scan the agents local folder structure for some means to identity the universal package downloaded.
